azure-pipelines/simple_test_framework.yml

```diff
@@ -27,43 +24,28 @@ jobs:
       maxParallel: 2
 
     steps:
+      - checkout: self
+
+      - checkout: git://TARDIS/tardis-refdata
```

Comment:
can we specify branches here?

Reply:
Yes. According to the documentation:

```
- checkout: git://MyProject/MyRepo@features/tools # checks out the features/tools branch
- checkout: git://MyProject/MyRepo@refs/heads/features/tools # also checks out the features/tools branch
- checkout: git://MyProject/MyRepo@refs/tags/MyTag # checks out the commit referenced by MyTag.
```

**Link:** https://docs.microsoft.com/en-us/azure/devops/pipelines/repos/multi-repo-checkout?view=azure-devops#checking-out-a-specific-ref